kratos/python/add_utilities_to_python.cpp

```diff
@@ -85,7 +89,9 @@ void AddUtilitiesToPython()
     ;
 
     class_<ConditionNumberUtility>("ConditionNumberUtility", init<>())
+    .def(init<LinearSolverType::Pointer, LinearSolverType::Pointer>())
     .def("GetConditionNumber",&ConditionNumberUtility::GetConditionNumber)
+    .def("GetConditionNumberInternally",&ConditionNumberUtility::GetConditionNumberInternally)
```

Comment:
What it means internally?

Reply:
It was the best name I could thought, means that uses the eigensolvers passed by the constructor instead of the used by calling the utility. If I give the same name is not possible to use it in python

Reply:
You can just overload the function in python like in Cpp. Nevertheless, there are some detail in importing overloaded method in [boost python](http://www.boost.org/doc/libs/1_61_0/libs/python/doc/html/tutorial/tutorial/functions.html#tutorial.functions.overloading)